javascript/modules/client/src/internal/interfaces/client.ts

```diff
@@ -82,6 +87,54 @@ export interface IWithdrawParams {
   tokenAddress?: string;
   reference?: string;
 }
+export interface IPermissionParams {
+  where: string;
+  who: string;
+  permission: DaoPermissions;
+}
+export interface IFreezeParams {
+  where: string;
+  permission: DaoPermissions;
+}
+
+export enum DaoPermissions {
+  UPGRADE_PERMISSION =
+    "0xd5a14b802580ed4dd25c37f4587d65235454a7d1c553c5226e9ae8a88921c15d",
+  SET_METADATA_PERMISSION =
```

Comment:
The hashed values are referred to as permission "ID". 
- `UPGRADE_PERMISSION_ID`